ome/omero-guide-upload
# Populate Metadata guide

Motivated by the interest in this workflow and https://github.com/ome/omero-guide-upload/issues/12, this PR is adding a new, completely UI-oriented chapter to the guide, with a separate sub-section specifically on SPW.


To test:
   - read the metadata and metadata-ui chapters
   - make sure they make sense, all links work and the cross-linking is working too
   - compare with the points in https://github.com/ome/omero-guide-upload/issues/12 and check this PR is solving them
 
cc @jburel @will-moore @sbesson @manics
